skyportal/handlers/api/classification.py

```diff
@@ -194,19 +199,18 @@ def post(self):
         data = self.get_json()
         obj_id = data['obj_id']
 
-        user_group_ids = [g.id for g in self.current_user.groups]
+        user_group_ids = [g.id for g in self.current_user.accessible_groups]
         group_ids = data.pop("group_ids", user_group_ids)
-        groups = Group.get_if_accessible_by(
-            group_ids, self.current_user, raise_if_none=True
-        )
 
         author = self.associated_user_object
 
         # check the taxonomy
         taxonomy_id = data["taxonomy_id"]
         taxonomy = Taxonomy.get_if_accessible_by(
-            taxonomy_id, self.current_user, raise_if_none=True
+            taxonomy_id, self.current_user, raise_if_none=False
```

Comment:
Is default value True or False?  If False, can remove this flag here and elsewhere.